wordpress-mobile/gutenberg-mobile
# Implement readable content margins

fixes #604 

This PR implements a JS only version of the iOS Readable Content Margins. This help us to show a narrower content view when the screens are too wide.

In the original ticket, @iamthomasbishop opted to keep the native iOS Readable Content Guide, but I opted for a simpler first iteration with a fixed maximum width. And later on implement platform specific details.

![ReadableContent](https://user-images.githubusercontent.com/9772967/54772906-b31d4500-4c08-11e9-8d2a-c22fbab90f04.png)

To test Android:
- Checkout this branch on WPAndroid project's `gutenberg-mobile` submodule.
- Run on a tablet (or tablet emulator)
- Check that the content is centered as in the image.
- Check that the selected blocks are boxed with a border.
- Run on a Phone
- Check that there are no side borders on selected blocks.

To test iOS:
- Check the steps mentioned here: https://github.com/wordpress-mobile/WordPress-iOS/pull/11316
